src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -409,46 +339,23 @@ private String getShardSampleName(final int shardIndex) {
     }
 
     /**
-     * Returns a list of {@link LocatableCopyNumberPosteriorDistribution} for {@link #sampleIndex} from a
+     * Returns a list of {@link IntervalCopyNumberGenotypingData} for {@link #sampleIndex} from a
```

Comment:
Hmm, I don't like conflating Posteriors and Data...I'm find with IntervalCopyNumberPosterior if PosteriorDistribution is too verbose.

Reply:
Ah, I see.  Is it because you store the baseline CN?  I'll leave it up to you.